cmd/mysterium_client/mysterium_client.go

```diff
@@ -18,40 +18,42 @@ func main() {
 		os.Exit(1)
 	}
 
-	cmdRun := client.NewCommand(options)
-	cmd.NewTerminator(cmdRun)
+	var cmdRun *client.Command
+	var cmdCli *cli.Command
+
+	stop := func() {
+		var killers []cmd.Killer
+		if cmdRun != nil {
+			killers = append(killers, cmdRun.Kill)
+		}
+		// TODO: add CLI killer once it's not blocking anymore
+		cmd.NewApplicationStopper(killers...)
+		os.Exit(0)
+	}
+
+	cmdRun = client.NewCommand(options, stop)
```

Comment:
cmdRun creator needs stop function which references cmdRun itself, that looks complicated :(

Reply:
Yes indeed - any suggestions?

Reply:
That's why sad face

Reply:
... Or we can try idea we talked about recently.

Reply:
This was simplified.

Reply:
👍